### PR TITLE
feat: add sticky scroll component

### PR DESCRIPTION
**Pull request by xdjeremy**

### CODE REVIEW

**Comment by xdjeremy** [comment]
* **#6** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/3?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/1?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @xdjeremy and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/services.astro
Comment (tamrivera05) [inline src/pages/services.astro:1]: are the comments necessary? kasi kung hindi remove mo na hmp >:(

**Review by tamrivera05 — CHANGES_REQUESTED** [review]
clean up! thumbs up ka pa din daw sabi ni George hehe
 Added via Giphy

**Comment by xdjeremy** [comment]
### Merge activity

* **May 4, 3:54 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/TimBuilding/Elysseum/3) rebased this pull request after merging its parent, because this pull request is set to merge when ready.
* **May 4, 10:37 PM EDT**: `xdjeremy` added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=TimBuilding&repo=Elysseum).
* **May 4, 10:41 PM EDT**: `xdjeremy`  merged this pull request with the [Graphite merge queue](https://app.graphite.dev/merges?org=TimBuilding&repo=Elysseum).

File: src/pages/services.astro
Comment (xdjeremy) [reply src/pages/services.astro:1]: k na

**Review by tamrivera05 — APPROVED** [review]

**Pull request merged**